include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -277,16 +277,21 @@ namespace VectorTools
               int spacedim,
               typename VectorType,
               template <int, int> class DoFHandlerType,
-              typename T>
+              typename T,
+              typename F>
```

Comment:
can you give this argument a more descriptive template type name? Maybe `FEorFECollectionType`?

Reply:
On the other hand, why do you need this? Regular `DoFHandler` has a function `get_fe_collection()` so that's a function you can generically call...

Reply:
@bangerth Right! I was happy that after pages of compiler errors it compiled ;)